lib/team.rb

```diff
@@ -0,0 +1,31 @@
+class Team
+  attr_reader :country, :players, :eliminated, :player_position,
```

Comment:
Be careful of your attr_readers!  You only need attr_readers for instance variables, not for other methods.  This should just be `attr_reader :country, :players` . The others should be methods defined in the class.